README.md

```diff
@@ -0,0 +1,16 @@
+# Hello World
+
+Set up a "Hello Microverse" project
+
+## Built With
+
+- html
+- css
+
+## Getting Started
+
+This is a startup project for "Set up a "Hello Microverse" project.
+
+## Authors
+
+PRATAP
```

Comment:
Please follow the README template and include the links to your professional social media profiles.